src/lib/tokens.ts

```diff
@@ -43,12 +43,13 @@ export const BREAKPOINT_OPTIONS = Object.keys(
 
 export const BREAKPOINTS = [
   ...Object.entries(designTokens.size.breakpoint),
-  ['base', 0],
-].map(([name, value]) => ({
+  ['base', {value: '0'}],
```

Comment:
@osbornm this branch is working, but there's a typescript error here and I'm not sure how to fix it.

🆘